src/pages/result.tsx

```diff
@@ -0,0 +1,145 @@
+import { Flex, Table, Title } from "@mantine/core";
+import React, { useState, useEffect } from "react";
+import "./result.css";
+
+type MatchTeam = {
+  left: {
+    id: string;
+    teamName: string;
+    isMultiWalk: boolean;
+    category: string;
+  };
+  right: {
+    id: string;
+    teamName: string;
+    isMultiWalk: boolean;
+    category: string;
+  };
+};
+type Result = {
+  left: {
+    teamID: string;
+    points: number;
+    time: number;
+  };
+  right: {
+    teamID: string;
+    points: number;
+    time: number;
+  };
+};
+type Match = {
+  id: string;
+  teams: MatchTeam;
+  matchType: string;
+  courseIndex: number;
+  results: Result;
+};
+
+export const Result = () => {
+  const [primarymatch, setprimaryMatch] = useState<Match[]>([]);
+  const [finalmatch, setfinalMatch] = useState<Match[]>([]);
+  useEffect(() => {
+    fetch(`${import.meta.env.VITE_API_URL}/match/primary`)
+      .then((res) => res.json())
+      .then((json) => setprimaryMatch(json))
+      .catch(() => alert("error"));
+  }, []);
+  // useEffect(() => {
+  //   fetch(`${import.meta.env.VITE_API_URL}/match/final`)
+  //     .then((res) => res.json())
+  //     .then((json) => setfinalMatch(json))
+  //     .catch(() => alert("error"));
+  // }, []);
```

Comment:
使わないコードは消してほしいです.   
消してToDoコメントを追加すべきかと
```suggestion
// ToDo: 本選の結果を取得する
```